visivo/commands/serve_phase.py

```diff
@@ -12,14 +12,22 @@
 VIEWER_PATH = pkg_resources.resource_filename("visivo", "viewer/")
 
 
-def get_project_json(output_dir):
+def get_project_json(output_dir, name_filter=None):
     project_json = ""
     with open(f"{output_dir}/project.json", "r") as f:
         project_json = json.load(f)
+    
+    if name_filter:
+        dashboards = [d for d in project_json['dashboards'] if d["name"] == name_filter]
```

Comment:
I wonder if there's way for us to read the project from the json back into a project object that has a dag so we can use the same filtering that you generated for the compile & run phase.

Reply:
We could do that for sure, might be a bit slower.

Reply:
Ah yeah, maybe after we address import performance that will be more practical

Reply:
Also, I just had some business questions about what would happen if you served a trace, for example